awx/main/utils/licensing.py

```diff
@@ -382,8 +383,15 @@ def validate(self):
 
         current_instances = Host.objects.active_count()
         license_date = int(attrs.get('license_date', 0) or 0)
-        automated_instances = HostMetric.objects.count()
-        first_host = HostMetric.objects.only('first_automation').order_by('first_automation').first()
+
+        model = getattr(settings, 'SUBSCRIPTION_USAGE_MODEL', '')
```

Comment:
When I first saw this variable I was like `Thats getting a setting, not  model?`. Could we rename this variable to something like `subscription_model` for easier clarity?